modules/core/src/main/java/org/apache/ignite/internal/processors/ru/RollingUpgradeStatus.java

```diff
@@ -30,33 +30,35 @@
      *
      * @return {@code true} if Rolling Upgrade is enabled.
      */
-    public boolean isEnabled();
+    public boolean enabled();
 
     /**
-     * @return {@code true} if strict mode is enabled.
+     * @return {@code true} if strict mode is disabled.
      */
-    public boolean isForcedModeEnabled();
+    public boolean forcedModeEnabled();
 
     /**
      * Returns the version that is used as starting point for Rolling Upgrade.
      *
      * @return Initial version.
      */
-    public IgniteProductVersion getInitialVersion();
+    public IgniteProductVersion initialVersion();
 
     /**
      * Returns the resulting version.
      * The returned value can be {@code null} if Rolling Upgrade is not in progress
      * or resulting version is not determined yet.
      *
+     * This method makes sense only for the case when the {@code forced} mode is disabled.
+     *
      * @return Resulting version.
      */
-    public @Nullable IgniteProductVersion getUpdateVersion();
+    public @Nullable IgniteProductVersion updateVersion();
```

Comment:
Name of this method sounds like it will update something.
Maybe updatingVersion / ongoingVersion / pendingVersion / smth else?

Reply:
What do you think about the following pair `initialVersion` & `targetVersion`?

Reply:
targetVersion sounds good.